src/types/opds1.ts

```diff
@@ -94,11 +96,29 @@ export const AccessRestrictionAudiobookMediaType =
 export const FindawayAudiobookMediaType =
   "application/vnd.librarysimplified.findaway.license+json";
 
+export const EPUB_MEDIA_TYPES = [
+  AxisNowWebpubMediaType,
+  EpubMediaType,
+  KepubMediaType
+];
+
 export type ReadOnlineMediaType =
   | typeof ExternalReaderMediaType
   | typeof AxisNowWebpubMediaType
   | typeof OverdriveEbookMediaType;
 
+export type EpubFormatType =
+  | typeof EpubMediaType
+  | typeof KepubMediaType
+  | typeof AxisNowWebpubMediaType;
```

Comment:
This works, but if you want `EpubFormatType` to track changes to `EPUB_MEDIA_TYPES` over time without manual updating, you can you something like this: 

```
  export type EpubFormatType = typeof EPUB_MEDIA_TYPES[number];
```